api/config/config.schema.ts

```diff
@@ -73,6 +73,7 @@ export default Joi.object({
   email: Joi.object({
     postmarkKey: Joi.string().required(),
     fromAddress: Joi.string().email().required(),
+    developers: Joi.array().items(Joi.string()).length(2),
```

Comment:
```suggestion
    developers: Joi.array().items(Joi.email()).length(2),
```

Reply:
If using this, what do you recommend for the defaults? Currently the defaults are not emails, so if the env is not set, postmark will silently error

Reply:
You can leave the defaults as `[]`, then if you `map` over the array to create the inputs for `sendBatch` instead of directly indexing it'll just pass an empty array to postmark (which should hopefully do nothing?)